### PR TITLE
Feature/new tab subframe

### PR DESCRIPTION
**Pull request by paul-pelith**
https://app.clickup.com/t/2ufkt8t

### CODE REVIEW

File: src/components/Common/TabGroup/styles.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import type { ThemeUICSSObject } from 'theme-ui';
+
+const tabActive: ThemeUICSSObject = {
```
Comment (ytwang00) [inline src/components/Common/TabGroup/styles.ts:3]: Why sometimes use Upper case on first case, sometimes use lower case?

File: src/components/StakingPage/index.tsx [hunk 2]
```diff
@@ -15,13 +15,27 @@ import { useTokenApprove, ApprovalState } from '../../hooks/useTokenApprove';
...
 import withConnectWalletCheckWrapper from '../../hoc/withConnectWalletCheckWrapper';
 import withApproveTokenCheckWrapper from '../../hoc/withApproveTokenCheckWrapper';
 import withWrongNetworkCheckWrapper from '../../hoc/withWrongNetworkCheckWrapper';
+import { TabGroup } from '../Common/TabGroup';
+
+import EthIcon from '../../images/icons/iconEthereum.svg';
+import SVGIcon from '../../images/icons/iconBSC.svg';
+import PolygonIcon from '../../images/icons/iconPolygon.svg';
+
+const TabsMocking = [
```
Comment (ytwang00) [inline src/components/StakingPage/index.tsx:34]: Why this is 'Mocking'? I think it's a config for Tabs, just call it config could be fine

**Review by ytwang00 — APPROVED** [review]

File: src/components/Common/TabGroup/styles.ts (hunk 1 above)
Comment (paul-pelith) [reply src/components/Common/TabGroup/styles.ts:3]: uppercase for react component and constant value (such as action name)
lower case for normal object value, this time is for css object

File: src/components/StakingPage/index.tsx (hunk 2 above)
Comment (paul-pelith) [reply src/components/StakingPage/index.tsx:34]: Will be modified when this integrate with correct action hooks

**Review by paul-pelith — COMMENTED** [review]
fix: naming

**Pull request merged**